common/hooks/pre-pkg/03-rewrite-python-shebang.sh

```diff
@@ -13,15 +13,28 @@ hook() {
 	fi
 
 	if [ -n "$pyver" ]; then
-		shebang="#!/usr/bin/python${pyver%.*}"
+		default_shebang="#!/usr/bin/python${pyver%.*}"
 	fi
 
 	find "${PKGDESTDIR}" -type f -print0 | \
 		while IFS= read -r -d '' file; do
 			[ ! -s "$file" ] && continue
-			[ -z "$(sed -n -E -e 2q -e '/^#!.*([[:space:]]|\/)python([0-9]\.[0-9])?([[:space:]]+|$)/p' "$file")" ] && continue
-			[ -n "$shebang" ] || msg_error "cannot convert shebang, set python_version\n"
-			echo "   Shebang converted to '$shebang': ${file#$PKGDESTDIR}"
+
+			pyinterp=$(sed -n -E -e 2q -e 's@^#!.*([[:space:]]|/)(python)([0-9](\.[0-9]+)?)?([[:space:]]+|$)@\2\3@p' "$file")
+			[ -z "$pyinterp" ] && continue
+
+			pyver=${pyinterp#*python}
```

Comment:
I don't think we need the `*` here, right?

Reply:
Right. The `*` was left over from earlier work that matched the shebang but didn't substitute. Thanks for catching this.